app/metrics.yaml

```diff
@@ -0,0 +1,87 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+$schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
+
+browser:
+  is_default:
+    type: boolean
+    lifetime: application
+    description: |
+      Is Focus the default browser? This is true only if the user
+      changes the default browser through the app settings.
+    bugs:
+      - https://github.com/mozilla-mobile/focus-android/issues/4545
+    data_reviews:
+      - https://github.com/mozilla-mobile/focus-android/issues/4901
```

Comment:
Note that this should be the link to a data-review response :-)

Reply:
I'll update this when I make a follow-up PR for the data review. Since the code here is not going to out to any release for now it should be fine to land the Glean integration code first.

Reply:
Sounds good!

Reply:
I've put the linked issue into our Focus refactoring project board so that we know to complete it before a release. :)